src/foxops/engine/rendering.py

```diff
@@ -14,6 +15,20 @@
 logger = get_logger(__name__)
 
 
+def first_ip_address_is_greater(first_ip, second_ip):
+    """Checks with of the two IP addresses is greater
+
+    Returns True if first one is greater. Otherwise False.
+    """
+    return ipaddress.ip_address(first_ip) > ipaddress.ip_address(second_ip)
+
+
+def get_next_ip(ip):
```

Comment:
should we make it more generic? I.e. call it `ip_add(ip: str, inc: int = 1)` and allow for arbitrary increases?